src/search_datastructures.hpp

```diff
@@ -517,9 +528,15 @@ class LocalDataHolder
     LocalDataHolder & operator=(LocalDataHolder &&) = delete;
 
     LocalDataHolder(LambdaOptions const & _options, TGlobalHolder & _gH) :
-        options{_options}, gH{_gH}, stats{},
-        searchScheme{search_schemes::expand(search_schemes::generator::pigeon_opt(0, options.maxSeedDist), options.seedLength)} // fixed search scheme
-    {}
+        options{_options}, gH{_gH}, searchOpts{options.searchOpts}, stats{},
+        searchScheme{search_schemes::expand(search_schemes::generator::pigeon_opt(0, options.searchOpts.maxSeedDist), options.searchOpts.seedLength)} // fixed search scheme
```

Comment:
This is probably a stupid question (apologies) but are these initialized before the function body is executed?

Reply:
Yep, they are guaranteed to be initialised before.

Reply:
But wouldn't we want the seed length from the search0 options if the search is iterative? Or am I missing something?

Reply:
That is set 3 lines further down in the file. Or what do you mean?

Reply:
Ah nevermind, this anyways changed now. Did not connect it. Now I get it with the updated version.